doc/source/whatsnew/v2.0.0.rst

```diff
@@ -1317,7 +1317,7 @@ Reshaping
 - Clarified error message in :func:`merge` when passing invalid ``validate`` option (:issue:`49417`)
 - Bug in :meth:`DataFrame.explode` raising ``ValueError`` on multiple columns with ``NaN`` values or empty lists (:issue:`46084`)
 - Bug in :meth:`DataFrame.transpose` with ``IntervalDtype`` column with ``timedelta64[ns]`` endpoints (:issue:`44917`)
-- Bug in :func:`agg` would ignore arguments when passed a list of functions (:issue:`50863`)
+- Bug in :meth:`DataFrame.agg`, :meth:`Series.agg`, :meth:`DataFrameGroupBy.agg`, :meth:`SeriesGroupBy.agg`, and :meth:`Resampler.agg` would ignore arguments when passed a list of functions (:issue:`50863`)
```

Comment:
I should have mentioned - this should be split up with the groupby and resampler part going under their section within Bugfixes.